src/startscene.js

```diff
@@ -0,0 +1,192 @@
+import {
+    MeshBuilder,
+    Color3,
+    Vector3,
+    Texture,
+    PointLight,
+    Scene,
+    StandardMaterial,
+    TrailMesh,
+    Scalar,
+    GlowLayer,
+    Animation,
+    ArcRotateCamera
+} from "@babylonjs/core";
```

Comment:
to enable tree shaking it is recommended to load directly from the directories instead of loading everything from @babylonjs/core. I know it sometimes feel like a hassle, but the package size will be thankful :-)

Reply:
oy, good to know; I'll have to revisit the tree shaking docs on some of this but I think I gotcha. It did make me wonder though -- since we're already using webpack, wouldn't a relatively simple (custom, obv) loader be able to take the above and transform it to the path syntax?